kueyen/metallb/ippool.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: metallb-system
 spec:
   addresses:
-  - 139.229.144.50-139.229.144.100
+  - 139.229.134.79-139.229.134.89
```

Comment:
This IP address range needs to be allocated in ipam.

Reply:
IPs are updated on IPAM 👍🏻